### DISCUSSION TITLE
home assistant camera api interface

### DISCUSSION THREAD

**Discussion by scooper1** [blakeblackshear/frigate#5339] [General]
is it possible to use a camera via the home assistant camera interface API ?

i have a camera in home assistant that is generated via an integration which I would like to use frigate to do object recognition

**Comment by NickM-27** [comment]
this is possible in 0.12 using go2rtc https://github.com/AlexxIT/go2rtc#source-hass

**Reply by scooper1** [reply]
any chance of a bit more details in using go2rtc to stream a hass camera to frigate

**Reply by NickM-27** [reply]
In frigate 0.12 go2rtc is built into frigate, you can follow the docs I linked for how to use that to configure the hass camera

**Reply by scooper1** [reply]
thanks  for the information, I got it to work
I did not spot that the link you gave had frigate information in it in a sub-link until I followed it fully

**Reply by scooper1** [reply]
I used the home assistant go2rtc in the end as was a bit simpler to configure
